MdePkg/Test/UnitTest/Library/BaseLib/BaseLibUnitTestApp.inf

```diff
@@ -1,33 +1,33 @@
-## @file
-# Unit tests of Base64 conversion APIs in BaseLib that are run from UEFI Shell.
-#
-# Copyright (C) Microsoft Corporation.
-# SPDX-License-Identifier: BSD-2-Clause-Patent
-##
-
-[Defines]
-  INF_VERSION                    = 0x00010006
-  BASE_NAME                      = BaseLibUnitTestsUefi
-  FILE_GUID                      = df5a6fed-8786-4a9d-9d02-eab39497b4a1
-  MODULE_TYPE                    = UEFI_APPLICATION
-  VERSION_STRING                 = 1.0
-  ENTRY_POINT                    = BaseLibUnitTestAppEntry
-
-#
-# The following information is for reference only and not required by the build tools.
-#
-#  VALID_ARCHITECTURES           = IA32 X64
-#
-
-[Sources]
-  Base64UnitTest.c
-
-[Packages]
-  MdePkg/MdePkg.dec
-
-[LibraryClasses]
-  BaseLib
-  BaseMemoryLib
-  UefiApplicationEntryPoint
-  DebugLib
-  UnitTestLib
+## @file
+# Unit tests of Base64 conversion APIs in BaseLib that are run from UEFI Shell.
```

Comment:
Did line endings change here? Can you ensure they are CRLF?

Reply:
Confirmed that they're CRLF.

Reply:
I pulled your branch locally and see LF. Can you please double check your autocrlf settings and try pulling directly from the remote branch?

![image](https://github.com/microsoft/mu_basecore/assets/21320094/4efa1c61-6169-4ac8-bf17-5d5a856b9c42)

Reply:
I had autocrlf set to true for my local repo.  Setting that to false allows me to see the line endings be LF.  I pushed a fix.